Packs/GenericSQL/Integrations/GenericSQL/README.md

```diff
@@ -0,0 +1 @@
+my
```

Comment:
please generate a readme.md with the `demisto-sdk generate-docs` command.
consult with @Bargenish if you need help in doing so.

please take special car with the integration general and command specific permissions.
are there any permissions the user should set in advance? try to be as detailed as possible.